### ISSUE TITLE
How to interpret the following output during the fit

### ISSUE THREAD

**Issue by cristobaltapia** [CamDavidsonPilon/lifelines#777]
Hi, 
when I fit an AFT model I get the following output during the fit process:
```
...
(23.71504157218443, None) True 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce49558d0>  
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce48ea7f0> 
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce48ea6a0> 
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce49558d0> 
(39.14159108151598, None) True 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce48ea7f0>  
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce48ea6a0> 
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce49558d0> 
(39.14159108151598, None) False 1.0 0.0 <lifelines.utils.DataframeSliceDict object at 0x7f5ce48ea7f0>
...
```

Is there something wrong? The model still converges, but it would be nice to know what this output means. From a fast reading of the code I couldn't figure it out. In this example I was trying the `WeibullAFTFitter`, but I get the same output with other custom models.

**Comment by CamDavidsonPilon** [comment]
Uh I think I left an errant print statement somewhere

**Comment by CamDavidsonPilon** [comment]
To confirm: you are using `robust=True`?

**Comment by cristobaltapia** [comment]
Yes. I was using that option.

**Comment by cristobaltapia** [comment]
So, I guess this was addressed by 155e9bb9e913e19858a91885a36ac844931c1eee, right? It works for me at least. Thanks!

**Comment by CamDavidsonPilon** [comment]
yea, but I should make a new release as well.
